engine/classes/Elgg/Database/UsersTable.php

```diff
@@ -566,32 +494,9 @@ public function setLastLogin(ElggUser $user) {
 			return;
 		}
 
-		$query = "
-			UPDATE {$this->table}
-			SET
-				prev_last_login = last_login,
-				last_login = :last_login
-			WHERE guid = :guid
-		";
-
-		$params = [
-			':last_login' => $time,
-			':guid' => (int) $user->guid,
-		];
-
 		$user->prev_last_login = $user->last_login;
 		$user->last_login = $time;
 
-		execute_delayed_write_query($query, null, $params);
```

Comment:
don't know... the running variable was already updated... the database now is instant updated... could only affect queries that run after this is updated...